src/components/TF2/Player/Player.tsx

```diff
@@ -189,6 +190,11 @@ const Player = ({
             // Immediately update local instance
             // Causes new info to immediately show
             player.localVerdict = e.toString();
+            color = displayColor(playerColors!, player, cheatersInLobby);

```

Comment:
Not sure why you need to reuse the `color` variable here. I tested what creating a new var would do and it appears to work fine.

Reply:
remnant from previous shenanigans